packages/web-react/src/components/Tooltip/useTooltip.ts

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import { ClickEvent } from '../../types';
+
+export interface UseTooltipProps {
+  isDismissible?: boolean;
+}
+
+export interface UseTooltipReturn {
+  open: boolean;
+  onClose: (event: ClickEvent) => void;
+}
+
+export const useTooltip = ({ isDismissible }: UseTooltipProps): UseTooltipReturn => {
+  const [open, setOpen] = useState<boolean>(isDismissible || false);
```

Comment:
`isDismissible` is not by default `false`?

Reply:
You are right, fixed